test/parallel/test-net-socket-destroy-send.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const common = require('../common');
+const net = require('net');
+const assert = require('assert');
+
+const server = net.createServer();
+server.listen(0, common.mustCall(function() {
```

Comment:
Personally I like it that if a `function` is used it's a strong indicator that you need `this` bound.